spec/defines/objects_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 facts = {
   :kernel   => 'Linux',
-  :os => {:family => 'Debian', :name => 'Debian'},
```

Comment:
This was a Bug, and only did not appear because `$::osfamily` was used in the PP-Files.